app/src/main/java/com/framgia/soundclound/screen/moretrack/MoreTrackFragment.java

```diff
@@ -0,0 +1,49 @@
+package com.framgia.soundclound.screen.moretrack;
+
+import android.annotation.SuppressLint;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.design.widget.BottomSheetDialogFragment;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.databinding.FragmentMoreTrackBinding;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+@SuppressLint("ValidFragment")
+public class MoreTrackFragment extends BottomSheetDialogFragment {
+
+    private Track mTrack;
+
+    @SuppressLint("ValidFragment")
+    public MoreTrackFragment(Track track) {
```

Comment:
2.11 Argument in Activity and Fragment
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md